OP_LOWERING_GUIDE.md

```diff
@@ -1,7 +1,7 @@
 # OP Lowering Guide
 
 ## Background
-PyTorch wraps the C++ ATen tensor library that offers a wide range of operations implemented on GPU and CPU. Pytorch/XLA is a PyTorch extension; one of its purposes is to convert PyTorch operations to XLA operations. Lowering defines a process of converting a higher-level representation to a lower-level representation. In this document, I will refer to the process of converting PyTorch operation to XLA operation as the lowering. XLA Compiler will also lower XlaOp to HLO, but that’s beyond the scope of this documentation. We will forward operations that we haven’t provided an XLA lowering yet to CPU and call ATen implementations. Operations that are forwarded to the CPU will cause a significant slowdown. We must lower all operations used in the model to achieve the best performance.
+PyTorch wraps the C++ ATen tensor library that offers a wide range of operations implemented on GPU and CPU. Pytorch/XLA is a PyTorch extension; one of its purposes is to convert PyTorch operations to XLA operations. Lowering defines a process of converting a higher-level representation to a lower-level representation. In this document, we will refer to the process of converting PyTorch operation to XLA operation as the lowering. The XLA compiler will also lower XlaOp to HLO, but that’s beyond the scope of this documentation. We will forward operations that we haven’t provided an XLA lowering yet to the CPU and call the efault ATen implementations. Operations that are forwarded to the CPU will cause a significant slowdown - we must lower all operations used in the model to achieve the best performance.
```

Comment:
I think you have a typo in  `call the default ATen implementations` (efault -> default)